public/styles/shared.css

```diff
@@ -7,8 +7,9 @@
 
   &:hover,
   &:focus-visible {
-    filter: blur(1px);
-    opacity: 1;
+    filter: grayscale(100%) !important;
+    opacity: 0.3 !important;
+    filter:blur(2) p !important;
```

Comment:
This must be wrong. First there must be a better way than `!important` to do this. Second you're defining `filter` twice so only the second one will apply anyway

Reply:
🤦 indeed, should have been  `filter:blur(2), grayscale(100%) !important;` mmm.... might have some unstagged stuff hanging